renderer/consts/keyboard-buttons.ts

```diff
@@ -0,0 +1,49 @@
+export const KeyboardButtons= {
```

Comment:
Let's make this `keyboardButtons`, usually first letter uppercase is for types or interfaces